tarantool/src/space.rs

```diff
@@ -317,6 +339,21 @@ impl Field {
         self
     }
 
+    /// Set the scale for the current field. Setting the scale explicitly
+    /// makes sense only if the field is a fixed point decimal. For other
+    /// fields it should keep the default value set by contructors. This
+    /// method captures `self` by value and returns it, so it should be
+    /// used in a builder fashion.
+    /// ```no_run
+    /// use tarantool::space::Field;
+    /// let f = Field::decimal128("middle name").with_scale(22);
+    /// ```
```

Comment:
```suggestion
    /// Set the scale for the current field.
    /// Setting the scale explicitly makes sense only if the field is a fixed point decimal. 
    /// For other fields it should keep the default value set by contructors. This
    /// method captures `self` by value and returns it, so it should be
    /// used in a builder fashion.
    ///
    /// ```no_run
    /// use tarantool::space::Field;
    /// let f = Field::decimal128("middle name").with_scale(22);
    /// ```
    ///
```

but it's a nitpick

Reply:
Let's keep the original style, which is also used in another similar function `Field::is_nullable` (see above in this file).